client/verta/verta/client.py

```diff
@@ -732,12 +743,91 @@ def set_repository(self, *args, **kwargs):
 
         """
         return self.get_or_create_repository(*args, **kwargs)
+    # set aliases for get-or-create functions for API compatibility
 
-    def get_or_create_registered_model(self, name=None, desc=None, tags=None, attrs=None, workspace=None, public_within_org=None, id=None):
-        raise NotImplementedError
+    def get_or_create_registered_model(self, *args, **kwargs):
+        """
+        Alias for :meth:`Client.set_project()`.
```

Comment:
Could we actually implement the logic here in `get_or_create_registered_model()`,
and then have `set_registered_model()` call this?

It will be different from the way `set_project()` etc are written, because we are trying to switch them.

Reply:
moved implementations